proposals/spring-2022.md

```diff
@@ -0,0 +1,49 @@
+# Telescope Spring 2022
+
+## Team
+- Paul Oppenheimer
+    - RPI Email: oppenp@rpi.edu
+    - <https://github.com/bepvte>
+    - bepvte@gmail.com
+- Zheyuan Chen
+    - RPI Email: chenz19@rpi.edu
+    - <https://github.com/ArberSephirotheca>
+
+## Links:
+- [repository](https://github.com/rcos/Telescope)
+- [website](https://telescope.rcos.io)
```

Comment:
Website should actually probably be `https://rcos.io` at this point, but telescope.rcos.io should redirect there so it's not really an issue.